avocado/plugins/xunit.py

```diff
@@ -63,15 +65,29 @@ def _create_failure_or_error(self, document, test, element_type):
         system_out = document.createElement('system-out')
         try:
             with open(test.get("logfile"), "r") as logfile_obj:
-                text_output = logfile_obj.read()
+                if max_log_size:
```

Comment:
The idiom `if max_log_size is not None` is preferred, and it achieves the goal of passing "0" and having no log file content in xunit.

Reply:
well there would be the `cut` line only in the output, but sure. I have nothing against "is not None"